### ISSUE TITLE
Multiple request interface

### ISSUE THREAD

**Issue by hadley** [r-lib/httr2#18]
List of requests + optional list of paths — retrieve in parallel.

**Comment by jennybc** [comment]
The Drive API batch request might be an interesting one to contemplate:

https://developers.google.com/drive/api/v3/performance#format-of-a-batch-request

Which then, of course, leads to the multiple response problem:

https://developers.google.com/drive/api/v3/performance#response-to-a-batch-request

So maybe there's also a question of who's doing the parallelization? httr2 vs. the API itself. Feel free to move this if it's in the wrong place.

**Comment by hadley** [comment]
That's a little different; here I'm thinking about performing multiple requests in parallel, not batching multiple requests into a single request.

**Comment by jennybc** [comment]
Do you think "batching multiple requests into a single request" should be its own issue? Or is this something that would fall in the domain of something more specialized like gargle? I assume Google isn't terribly unique in having such a thing but I'm speculating. I find the response processing more difficult to think about than request formation.

**Comment by hadley** [comment]
I think it's unique to gargle. I think it's pretty uncommon outside of google apis, and I doubt there is any standard for this sort of request batching.

**Comment by jennybc** [comment]
OK. Feel free to delete hide this stuff then. Maybe implementing batched requests/responses in gargle would be a good test of httr2 at some point.
